stetho-sample/src/debug/java/com/facebook/stetho/sample/SampleDebugApplication.java

```diff
@@ -28,10 +36,33 @@ public void onCreate() {
 
     long startTime = SystemClock.elapsedRealtime();
     final Context context = this;
+
+    // Setup a javascript runtime
```

Comment:
Err, accidentally commented on the diff previously.  Let's try to simplify this boilerplate (possibly with an Android-specific helper?)